### DISCUSSION TITLE
ScriptReference/Callbacks.DidReloadScripts

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#12] [Notes]
# ScriptReference/Callbacks.DidReloadScripts

https://docs.unity3d.com/ScriptReference/Callbacks.DidReloadScripts.html

**Comment by vertxxyz** [comment]
Related callbacks can be found in [`AssemblyReloadEvents`](https://docs.unity3d.com/ScriptReference/AssemblyReloadEvents.html).
